src/Core/Services/Implementations/CipherService.cs

```diff
@@ -130,9 +130,8 @@ public class CipherService : ICipherService
             else
             {
                 // Make sure the user can save new ciphers to their personal vault
-                var personalOwnershipPolicyCount = await _policyRepository.GetCountByTypeApplicableToUserIdAsync(savingUserId,
-                    PolicyType.PersonalOwnership);
-                if (personalOwnershipPolicyCount > 0)
+                var personalOwnershipPolicies = await _policyService.GetPoliciesApplicableToUserAsync(savingUserId, PolicyType.PersonalOwnership);
+                if (personalOwnershipPolicies.Any())
```

Comment:
This is a common pattern, how about adding another interface that just returns a `bool` if the policy applies? Then use that for any caller that currently has a `policies.Any()` test.